storybook/src/docs/form/components/FieldContainer.stories.tsx

```diff
@@ -35,6 +36,16 @@ export const Basic = {
                 <FieldContainer label="Helper" helperText="This is a helper">
                     <InputBase onChange={handleChange} value={value} placeholder="Placeholder" />
                 </FieldContainer>
+                <br />
+                <FieldContainer label="Secondary Helper Text" secondaryHelperText="0/100">
+                    <InputBase onChange={handleChange} value={value} placeholder="Placeholder" />
+                </FieldContainer>
+                <FieldContainer label="Multiple Helper Texts" helperText="Helper Text" secondaryHelperText="0/100">
+                    <InputBase onChange={handleChange} value={value} placeholder="Placeholder" />
+                </FieldContainer>
```

Comment:
Could you add the actual character-count of the input (instead of hardcoding `0`) to the `secondaryHelperText` to demonstrate an actual use-case?